pontoon/base/models/entity.py

```diff
@@ -843,20 +844,29 @@ def for_project_locale(
                 entities = entities.distinct()
 
         # TODO: Uncomment the following lines to reactivate the
-        #       feature once all search options are implemented:
-        #       - 857 (rejected translations)
-        #       - 869-870 (context identifiers)
-        #       - 883-884 (translations only)
+        #       context identifiers filter once .ftl bug is fixed (issue #3284):
+        #       - 883-888
```

Comment:
Now it's probably best if you move this comment where these lines are, otherwise you'll need to keep the line numbers accurate. :)